source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -100,6 +100,10 @@ class AuthenticatorImpl : public Logger::Loggable<Logger::Id::jwt>,
   const bool is_allow_failed_;
   const bool is_allow_missing_;
   TimeSource& time_source_;
+
```

Comment:
OK,  I am fine to add 5 second slack time for expiration checking.

Here are some tips for the code cleanup